doc/nrf/known_issues.rst

```diff
@@ -9,6 +9,10 @@ Known issues
    :local:
    :depth: 2
 
+.. note::
+
+   You are looking at the page for the patch release. For the latest version of the page that tracks updates on the ``main`` branch, see :ref:`known_issues`.
```

Comment:
Shouldn't it be `known issues for nRF Connect SDK v2.1.3`_  rather than :ref:`known_issues`?
```suggestion
   You are looking at the page for the patch release. For the latest version of the page that tracks updates on the ``main`` branch, see :ref:`known_issues`.
```

Reply:
Not sure about this.